kymatio/scattering1d/backend/jax_backend.py

```diff
@@ -0,0 +1,8 @@
+from ...backend.jax_backend import JaxBackend
+from .numpy_backend import NumpyBackend1D
+
+
+class JaxBackend1D(JaxBackend, NumpyBackend1D):
```

Comment:
That is a good point.

Here is some pictorial evidence that it works:

![image](https://user-images.githubusercontent.com/1306635/124051478-9462a700-d9ea-11eb-8c01-0e17b00a1156.png)

I'll leave this comment open because I haven't changed this yet, but probably will to enhance readability.

Reply:
I was about to comment on how I only added a comment because I thought that adding `_np` and `_fft` manually would be confusing because of it already being in `JaxBackend`. But your suggestion is to remove `JaxBackend`.

I see. That is definitely worth considering.

Reply:
That is correct. I'll leave it up to you to decide.

Reply:
@eickenberg make a decision?